mantidproject/mantid
# Stop listing EFixed for each detector pixel in BASIS instrument

**Description of work.**
- [x] new instrument file that does not list `EFixed` for each detector
- [x] simpler names for previous IDF's

**Report to:** N/A

**To test:**
Run this script in the latest mantid release:
```
BASISReduction(RunNumbers='55555', MonitorNorm=True, ReflectionType='silicon111', EnergyBins=[-100, 0.4, 100], MomentumTransferBins=[0.3, 0.2, 1.9])
SaveNexus(InputWorkspace='BSS_55555_sqw', Filename='/tmp/old_spectra.nxs')
```
Run this script in the compiled mantid from this branch:
```
old_spectra = LoadNexus(Filename='/tmp/old_spectra.nxs')
BASISReduction(RunNumbers='55555', MonitorNorm=True, ReflectionType='silicon111', EnergyBins=[-100, 0.4, 100], MomentumTransferBins=[0.3, 0.2, 1.9])
w =Min(old_spectra)
tolerance = 0.01 * min([w.dataY(i)[0] for i in range(w.getNumberHistograms())])
CompareWorkspaces(Workspace1='BSS_55555_sqw', Workspace2='old_spectra', Tolerance=tolerance, CheckInstrument=False)
```
Verify that `CompareWorkspaces` is sucessful

Fixes #24737

This does not require release notes because it's a backend change with no change in user interface

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
